pkg/util/utils.go

```diff
@@ -69,6 +69,32 @@ func StringInSlice(s string, sl []string) bool {
 	return false
 }
 
+// ParseChanges returns key, value(s) pair for given option.
+func ParseChanges(option string) (key string, vals []string) {
+	// Supported format as below
+	// 1. key=value
+	// 2. key value
+	// 3. key ["value","value1"]
+	if strings.Contains(option, " ") {
+		// This handles 2 & 3 conditions.
+		var val string
+		tokens := strings.SplitAfterN(option, " ", 2)
+		key = strings.Trim(tokens[0], " ") // Need to trim whitespace part of delimeter.
+		val = tokens[1]
+		if strings.Contains(tokens[1], "[") {
+			//Trim '[',']' if exist.
+			val = strings.TrimLeft(strings.TrimRight(tokens[1], "]"), "[")
+		}
+		vals = strings.Split(val, ",")
+	} else if strings.Contains(option, "=") {
+		// handles condition 1.
+		tokens := strings.Split(option, "=")
```

Comment:
`SplitN(option, "=", 2)` probably safer

Reply:
In case of parameters like ``KEY=Opt=val2``, This will not work or we need one more split.